src/data_structures/src/array_ext.cairo

```diff
@@ -43,6 +44,7 @@ trait SpanTraitExt<T> {
     fn index_of_max<impl TPartialEq: PartialEq<T>, impl TPartialOrd: PartialOrd<T>>(
         self: Span<T>
     ) -> Option<usize>;
+    fn dedup<impl TPartialEq: PartialEq<T>>(self: Span<T>) -> Array<T>;
```

Comment:
can you add documentation on what the function does plz

Reply:
Where should i add that

Reply:
right above the function, rust style doc